http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/Boot.scala

```diff
@@ -145,7 +145,7 @@ object Boot extends IOApp {
         dataprocConfig,
         gceConfig
       )
-      val runtimeService = new RuntimeServiceInterp[IO](
+      val runtimeService = RuntimeService(
```

Comment:
Should `RuntimeServiceInterp` be made private given we have this constructor?

Reply:
yea, I tried..but we're testing some members of `RuntimeServiceInterp` directly in unit tests, it's not as easy to make it private